robot/Cumulus/resources/NPSP.py

```diff
@@ -51,6 +52,9 @@ def _init_locators(self):
             response = client._call_salesforce(
                 'GET', 'https://{}/services/data'.format(client.sf_instance))
             latest_api_version = float(response.json()[-1]['version'])
+            if not latest_api_version in locators_by_api_version:
+                warnings.warn("Could not find locator library for API %d" % latest_api_version)
```

Comment:
@prescod I asked for using `self.builtin.log()` because that will end up in the Robot Framework log output which is presumably where people will be looking to debug robot tests.

@boakley let me know if I'm mistaken

Reply:
I tested it and it doesn't seem to end up in any file. Most likely because it happens before the testing has started.

Reply:
Oh, that seems plausible. Okay, let's leave it for now.